source/develop/release-management/features/network/unrestricted-network-names.html.md

```diff
@@ -71,8 +74,8 @@ This would affect anywhere a network name shows up, and in particular:
 
 Currently, if a user has a connectivity issue regarding a network FOO on one of his hosts, he can easily look FOO up in Engine. Now this would become more difficult, as the host is no longer exposed to a human-legible name. We should supply means to alleviate this:
 
-1.  ovirt-engine-shell should be able to list the `brXXXXXXXXXXXXX` name per each network. A script built upon it (using grep, possibly) would be able to map a `brXXXXXXXXXXXXX` name into the visible name of its owning network.
-2.  Make it possible to look a network up based on its vdsm_name (`brXXXXXXXXXXXXX`) in the network tab and in via the search box
+1.  ovirt-engine-shell should be able to list the `onXXXXXXXXXXXXX` name per each network. A script built upon it (using grep, possibly) would be able to map a `onXXXXXXXXXXXXX` name into the visible name of its owning network.
+2.  Make it possible to look a network up based on its vdsm_name (`onXXXXXXXXXXXXX`) in the network tab and in via the search box
```

Comment:
Suggestion:
"Make it possible to look up a network in the network tab and via the search box, based on its vdsm_name (onxxxxxxxxxxxxx)."